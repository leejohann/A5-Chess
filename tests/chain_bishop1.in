move 4 6 4 5
move 3 1 3 2
move 5 7 4 6
move 2 0 3 1
move 4 6 7 3
move 3 1 0 4
quit
