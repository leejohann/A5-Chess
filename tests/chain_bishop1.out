RNBQKBNR
PPPPPPPP
 _ _ _ _
_ _ _ _ 
 _ _ _ _
_ _ _ _ 
pppppppp
rnbqkbnr

it is White's turn
RNBQKBNR
PPPPPPPP
 _ _ _ _
_ _ _ _ 
 _ _ _ _
_ _ p _ 
pppp ppp
rnbqkbnr

it is Black's turn
RNBQKBNR
PPP PPPP
 _ P _ _
_ _ _ _ 
 _ _ _ _
_ _ p _ 
pppp ppp
rnbqkbnr

it is White's turn
RNBQKBNR
PPP PPPP
 _ P _ _
_ _ _ _ 
 _ _ _ _
_ _ p _ 
ppppbppp
rnbqk nr

it is Black's turn
RN QKBNR
PPPBPPPP
 _ P _ _
_ _ _ _ 
 _ _ _ _
_ _ p _ 
ppppbppp
rnbqk nr

it is White's turn
RN QKBNR
PPPBPPPP
 _ P _ _
_ _ _ _b
 _ _ _ _
_ _ p _ 
pppp ppp
rnbqk nr

it is Black's turn
RN QKBNR
PPP PPPP
 _ P _ _
_ _ _ _b
B_ _ _ _
_ _ p _ 
pppp ppp
rnbqk nr

it is White's turn

